awscli/examples/application-signals/delete-service-level-objective.rst

```diff
@@ -0,0 +1,12 @@
+**To delete the specified service level objective.**
+
+The following ``delete-service-level-objective`` example deletes the specified service level objective. ::
+
+    aws application-signals delete-service-level-objective \
+    --id "arn:aws:application-signals:us-east-1:123456789101:slo/SLOName"
+
+Output::
```

Comment:
```suggestion
```

Reply:
Can you remove the `Output::` line from the commands that don't have any. It's actually invalid .rst, hence the test failures.